Source/WebCore/loader/DocumentLoader.cpp

```diff
@@ -2535,6 +2535,17 @@ void DocumentLoader::contentFilterHandleProvisionalLoadFailure(const ResourceErr
 
 #endif // ENABLE(CONTENT_FILTERING)
 
+bool DocumentLoader::isDisabledContentRuleListIdentifier(const String& identifier) const
+{
+    return m_disabledContentRuleListIdentifiers.contains(identifier);
+}
+
+void DocumentLoader::setDisabledContentRuleListIdentifiers(const Vector<String>& identifiers)
+{
+    m_disabledContentRuleListIdentifiers.clear();
```

Comment:
This should be m_disabledContentRuleListIdentifiers = WTFMove(identifiers)

Reply:
So `m_disabledContentRuleListIdentifiers` is a `HashMap`, while `identifiers` is a `Vector` (I opted to make `identifiers` a vector in the policies/IPC plumbing, to avoid some unnecessary hashing when propagating that list).

Are you suggesting that I should make this a `HashMap` throughout the code?